presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTaskFactory.java

```diff
@@ -164,6 +168,13 @@ public ThreadPoolExecutorMBean getExecutor()
         return executorMBean;
     }
 
+    @Managed
+    @Nested
```

Comment:
I don't think we need `@Nested` here since this is a primitive type.

Reply:
Oh I thought to return it as DecayCounter, my mistake 😄